proto/cmp/services/accommodation/v1alpha1/search_parameters_types.proto

```diff
@@ -39,8 +40,10 @@ message AccommodationSearchParameters {
   repeated cmp.types.v1alpha1.RatePlan rate_plan = 7;
 
   // Rate Rules
-  // FIXME: Did we decided to remove this from SearchParameters?
-  // FIXME: Maybe some users/distributors would want to search for only
-  // FIXME: RESELLABLE offers?
+  // To be used when searching for specific rates like refundable or resellable offers
   repeated cmp.types.v1alpha1.RateRule rate_rules = 8;
-}
+
+  // Product code list
+  // Here a list of property codes would be used
+  repeated cmp.types.v1alpha1.ProductList product_list = 9;
```

Comment:
To be in line with the other comments this part should also be updated as this:

```suggestion
  repeated cmp.types.v1alpha1.ProductCode product_codes = 9;
```

Reply:
Agreed and implemented